src/niamoto/core/plugins/transformers/class_objects/series_matrix_extractor.py

```
@@ -58,120 +54,164 @@
-        if not series:
+        try:
+            validated_config = self.config_model(**config)
+            return validated_config
+        except ValidationError as e:
+            plugin_name = config.get("plugin", "class_object_series_matrix_extractor")
             raise DataTransformError(
-                "At least one series must be specified",
-                details={"config": config},
+                f"Invalid configuration for {plugin_name}: {e}",
+                details={"pydantic_error": e.errors(), "config": config},
             )
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)